src/hera/cron_workflow.py

```diff
@@ -96,6 +96,8 @@ def build(self) -> IoArgoprojWorkflowV1alpha1CronWorkflow:
             setattr(cron_workflow_spec, "timezone", self.timezone)
 
         return IoArgoprojWorkflowV1alpha1CronWorkflow(
+            api_version="argoproj.io/v1alpha1",
+            kind="CronWorkflow",
```

Comment:
I would suggest a package level constant `API_VERSION = "argoproj.io/v1alpha1"`.  And for `kind` downstream we considered using `workflow.__class__.__name__` but this might be a bit brittle, so a `self.kind` class member variable would be great.

Reply:
Thank you for the input @elliotgunton! Perhaps we can set the API version the [same way we set the global host and token](https://github.com/argoproj-labs/hera-workflows/blob/main/src/hera/host_config.py)? I think the thought of using `workflow.__class__.__name__` is great as long as Hera will exclusively use class names that are conceptually equivalent to Argo Workflows. So in the context of `Workflow`, `WorkflowTemplate`, and `CronWorkflow` this holds. Perhaps we can start with your suggestion of using the class name and then introduce a `kind` concept later if need be?